python/py-twisted/Portfile

```diff
@@ -39,6 +39,8 @@ if {${name} ne ${subport}} {
         port:py${python.version}-incremental \
         port:py${python.version}-hyperlink \
         port:py${python.version}-constantly
+    depends_run-append \
+        port:py${python.version}-pyhamcrest
```

Comment:
I think this should be added to depends_lib as it's listed in install_requires, so importing twisted will use PyHamcrest, too.

Reply:
I don't quite understand what you are trying to say here. PyHamcrest doesn't seem to be needed during installation, so I'm not sure what we would gain by adding it to `depends_lib`. Whenever Twisted is used, PyHamcrest should be active given these rules. What am I missing?